sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/CosmosAsyncContainer.java

```diff
@@ -302,6 +334,9 @@ public String getId() {
      */
     public <T> CosmosPagedFlux<T> readAllItems(FeedOptions options, Class<T> classType) {
         return UtilBridgeInternal.createCosmosPagedFlux(pagedFluxOptions -> {
+            String spanName = "readAllItems." + this.getId();
```

Comment:
id is the id of the container. so it means in the scope of this container "readAllItems." + this.getId() can be pre-computed for the container, rather than re-computing it each time.

Could you pre-compute this in the container level.

here and other places.